specification/databox/resource-manager/Microsoft.DataBox/preview/2024-03-01-preview/databox.json

```diff
@@ -0,0 +1,7264 @@
+          "description": "Postal code.",
+          "type": "string"
+        },
+        "zipExtendedCode": {
+          "description": "Extended Zip Code.",
+          "type": "string"
+        },
+        "companyName": {
+          "description": "Name of the company.",
+          "type": "string"
+        },
+        "addressType": {
+          "description": "Type of address.",
+          "enum": [
+            "None",
+            "Residential",
+            "Commercial"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AddressType",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "None",
+                "description": "Address type not known."
+              },
+              {
+                "value": "Residential",
+                "description": "Residential Address."
+              },
+              {
+                "value": "Commercial",
+                "description": "Commercial Address."
+              }
+            ]
+          },
+          "default": "None"
+        },
+        "skipAddressValidation": {
+          "type": "boolean",
+          "description": "Flag to indicate if customer has chosen to skip default address validation"
+        },
+        "taxIdentificationNumber": {
+          "type": "string",
+          "description": "Tax Identification Number"
+        }
+      }
+    },
+    "Sku": {
+      "description": "The Sku.",
+      "required": [
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The sku name.",
+          "enum": [
+            "DataBox",
+            "DataBoxDisk",
+            "DataBoxHeavy",
+            "DataBoxCustomerDisk"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "DataBox",
+                "description": "Data Box."
+              },
+              {
+                "value": "DataBoxDisk",
+                "description": "Data Box Disk."
+              },
+              {
+                "value": "DataBoxHeavy",
+                "description": "Data Box Heavy."
+              },
+              {
+                "value": "DataBoxCustomerDisk",
+                "description": "Data Box Customer Disk."
+              }
+            ]
+          }
+        },
+        "displayName": {
+          "description": "The display name of the sku.",
+          "type": "string"
+        },
+        "family": {
+          "description": "The sku family.",
+          "type": "string"
+        },
+        "model": {
+          "enum": [
+              "DataBox",
+              "DataBoxDisk",
+              "DataBoxHeavy",
+              "DataBoxCustomerDisk",
+              "DataBoxV2x100",
+              "DataBoxV2x500"
+          ],
+          "type": "integer",
```

Comment:
Is it a typo? I guess the type should be "string"?

Reply:
Updated it to string